software/cloud-dashboard/node-server/config/device-map.js

```diff
@@ -6,7 +6,7 @@
 module.exports={
 	locMap : {
 	"4b0031000d51353432393339": "ADPLKenyaC6304",
-        "300057000951343334363138": "ADPLKenyaN3763",
+        "3e001d000e51353432393339": "ADPLKenyaN6833",
```

Comment:
This device is the same site as the 3763, correct? The actual mapping to name can be arbitrary, and does not need to reflect the particle's actual "name"

If we keep it `ADPLKenyaN3763` or just change it to something more generic like `ADPLKenya_1` we can keep the database data grouped together for that site and still identify the actual particle based on the coreid :).

What're your thoughts @aforbis-stokes?